modules/vcf2maf/1.3/vcf2maf.smk

```diff
@@ -83,8 +107,8 @@ rule _vcf2maf_run:
         stderr = CFG["logs"]["vcf2maf"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}_vcf2maf.stderr.log",
     params:
         opts = CFG["options"]["vcf2maf"],
-        build = lambda w: VCF2MAF_GENOME_VERSION_MAP[w.genome_build],
-        custom_enst = op.switch_on_wildcard("genome_build", CFG["switches"]["custom_enst"])
+        build = lambda w: VCF2MAF_VERSION_MAP[w.genome_build],
+        custom_enst = lambda w: config['lcr-modules']["vcf2maf"]["switches"]["custom_enst"][VCF2MAF_GENOME_VERSION[w.genome_build]]
```

Comment:
I am not sure I follow how this is tracking with module config and switches. There is no key for example for `grch38` in config. Are these updates tested/all updates are pushed?

Reply:
I changed the vcf2maf config in gambl, but not in the module itself. Will update